src/tasks/task2.js

```diff
@@ -0,0 +1,72 @@
+import fs from 'fs';
+import csv from 'csvtojson';
+import * as logger from '../../services/logger';
+
+const _filePath = './scv/node_mentoring_t1_2_input_example.csv';
+const _reader = fs.createReadStream(_filePath);
+const _writer = fs.createWriteStream('./task2.txt');
+
+const _convertoToJson = chunk => {
+  return csv()
+    .fromString(chunk)
+    .then(data => data);
+}
+
+const _slicedData = data => data.slice(1, data.length);
```

Comment:
using underscore for module scope variables is not necessary, because you can use export to mark properties you want to share.  It is more popular to use this convention to mark private variables inside class.

Reply:
Replaced.